docs/src/test-advanced.md

```diff
@@ -467,16 +467,9 @@ For TypeScript, also add the following to `global.d.ts`. You don't need it for J
 
 ```js
 // global.d.ts
-declare namespace folio {
+declare namespace PlaywrightTest {
   interface Matchers<R> {
     toBeWithinRange(a: number, b: number): R;
   }
 }
 ```
-
-To import expect matching libraries like [jest-extended](https://github.com/jest-community/jest-extended#installation) you can import it from your `global.d.ts`:
```

Comment:
Wouldn't something like that work for users then? (move extending from jest from us to the use-space). Then the native expect-extend imports should work out of the box.

```ts
declare namespace PlaywrightTest {
  interface Matchers<R> extends jest.Matchers<R> {
  }
}
```